tests/nexus/guest/src/main.rs

```diff
@@ -0,0 +1,8 @@
+#![cfg_attr(target_arch = "riscv32", no_std, no_main)]
+
+use nexus_rt::print;
+
+#[nexus_rt::main]
+fn main() {
+    print!("Hello, World!\n");
```

Comment:
Would be great to have a program that has inputs and it commits to some data as public input